### PR TITLE
Adds 12_map_integration large map

### PR DESCRIPTION
**Pull request by francocipollone**
# 🎉 New feature

image

## Summary
 - Create a large map (XODR) using the 12 submaps from resources.
 - The geoJSON is also provided with the information of the crosswalks.
 - README was added for explaining the procedure when creating the integration map.

## Test it
 - For xodr file
    Verify the file using https://odrviewer.io/ or [maliput_viz](https://github.com/maliput/maliput_viz) app
 - For geoJSON file
    Verify the geoJSON file with https://geojson.io/

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if it affects the public API)

### CODE REVIEW

File: standalone/12_map_integration/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+```
+
+sub_map_1 ----- sub_map_2 ----- sub_map_3 ----- sub_map_4 ----- sub_map_5 ----- sub_map_6
+    |
+    |
+    |
+sub_map_7 ----- sub_map_8 ----- sub_map_9 ----- sub_map_10 ----- sub_map_11 ----- sub_map_12
+```
+
+ - The submaps aren't connected each other.
```
Comment (liangfok) [inline standalone/12_map_integration/README.md:38]: Missing "to" after "connected".

File: README.md [hunk 2]
```diff
@@ -57,3 +57,11 @@ WIDTH=4 RADIUS=8 CROSSWALK=True CROSSWALK_LENGTH=2 empy3 templates/intersection.
 ## Resources
 
 Example files with different values are provided under the `resources` folder. 
+
+## Standalone maps
+
+Standalone maps that doesn't respond to a particular template are added in the `standalone` folder.
```
Comment (liangfok) [inline README.md:63]: "doesn't" should be "don't" or "do not".

"respond" should be "correspond".

File: README.md [hunk 3]
```diff
@@ -57,3 +57,11 @@ WIDTH=4 RADIUS=8 CROSSWALK=True CROSSWALK_LENGTH=2 empy3 templates/intersection.
...
+
+ - [12_map_integration](standalone/12_map_integration/) map: A large-combined map was created out of the available `resources`. See [12_map_integration/README](standalone/12_map_integration/README.md) for more information about the map creation.
```
Comment (liangfok) [inline README.md:65]: Should add "process" after "creation".

Should also wrap this line at 120 characters.

File: standalone/12_map_integration/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+### Procedure for creating the large map
+
+For creating the map the following steps were taken.
+
+1. Create the 12 submaps with the correspondent offset in a way that when merged no overlapping occurs
+2. Adapt the road ids and junction ids to avoid any kind of clashing among ids.
+3. Merge all OpenDRIVE description into one.
```
Comment (liangfok) [inline standalone/12_map_integration/README.md:46]: "description" should be "descriptions". Ditto on the line below.

**Review by liangfok — CHANGES_REQUESTED** [review]
Completed first pass.

File: standalone/12_map_integration/README.md (hunk 4 above)
Comment (francocipollone) [reply standalone/12_map_integration/README.md]: Done

File: README.md (hunk 3 above)
Comment (francocipollone) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (francocipollone) [reply README.md]: Oh my bad. Done

File: standalone/12_map_integration/README.md (hunk 1 above)
Comment (francocipollone) [reply standalone/12_map_integration/README.md]: Done

File: standalone/12_map_integration/README.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
+## 12_map_integration map
+
+
+The 12_map_integration map is composed by 12 submaps.
```
Comment (liangfok) [inline standalone/12_map_integration/README.md:4]: "by" should be "of".

**Review by liangfok — CHANGES_REQUESTED** [review]
One final polishing fix.

File: standalone/12_map_integration/README.md (hunk 5 above)
Comment (francocipollone) [reply standalone/12_map_integration/README.md]: Addressed

**Comment by francocipollone** [comment]
I will proceed and merge it

**Pull request merged**